test/dyndns-test.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<opnsense>
+  <system>
+  </system>
+  <OPNsense>
```

Comment:
again: it seems you did not understand thze scope of a test
your starting and expected xml are identic :-( what do you test here?
```diff
diff --git a/test/dyndns-test.xml b/test/dyndns-test.xml
index 16ad005..38af667 100644
--- a/test/dyndns-test.xml
+++ b/test/dyndns-test.xml
@@ -3,34 +3,5 @@
   <system>
   </system>
   <OPNsense>
-    <DynDNS>
-      <general>
-        <enabled>1</enabled>
-        <verbose>0</verbose>
-        <allowipv6nano>0</allowipv6nano>
-        <daemon_delay>300</daemon_delay>
-        <backend>opnsense</backend>
-      </general>
-      <accounts>
-        <account uuid="8e4627c4-21ff-4252-a331-3d1adee0a023">
-          <enabled>1</enabled>
-          <service>noip</service>
-          <protocol/>
-          <server/>
-          <username/>
-          <password>user</password>
-          <resourceId>pass</resourceId>
-          <hostnames>all.ddnskey.com</hostnames>
-          <wildcard>0</wildcard>
-          <zone/>
-          <checkip>web_noip-ipv4</checkip>
-          <checkip_timeout>10</checkip_timeout>
-          <force_ssl>1</force_ssl>
-          <ttl>300</ttl>
-          <interface>wan</interface>
-          <description>dyndns-description</description>
-        </account>
-      </accounts>
-    </DynDNS>
   </OPNsense>
 </opnsense>
```